app/views/media_objects/_resource_description.html.erb

```diff
@@ -18,12 +18,12 @@ Unless required by applicable law or agreed to in writing, software distributed
 <%= form_for @mediaobject, html: { class: 'form-vertical' } do |form| %>
   <%= hidden_field_tag :step, @active_step %>
   
-  <%#= render partial: 'text_field',
+  <%= render partial: 'text_field',
               locals: {form: form, field: :bibliographic_id,
                       options: {display_label: 'Bibliographic ID',
                                 dropdown_field: :bibliographic_id_label, 
                                 dropdown_options: ModsDocument::IDENTIFIER_TYPES,
-                                extra_classes: Avalon::BibRetriever.configured? ? 'import-button' : nil}} #%>
+                                extra_classes: Avalon::BibRetriever.configured? ? 'import-button' : nil}} %>
```

Comment:
is there a reason to set this to nil if not configured? why not just not set it in this case?
```ruby
extra_classes: 'import-button' if Avalon::BibRetriever.configured?
```